pandas/tests/window/test_rolling_skew_kurt.py

```diff
@@ -219,12 +221,14 @@ def test_rolling_kurt_edge_cases(step):
 
 
 def test_rolling_skew_eq_value_fperr(step):
-    # #18804 all rolling skew for all equal values should return Nan
```

Comment:
Could you keep these comments referencing old issues? Helps maintain history of the changed test